apps/projects/app/context/DecoratedRepos.js

```diff
@@ -1,13 +1,25 @@
 import React from 'react'
 import { useAragonApi } from '../api-react'
 import { GraphQLClient } from 'graphql-request'
+import { ipfsGet } from '../utils/ipfs-helpers'
 
 const repoQuery = repoId => `{
   node(id: "${repoId}") {
     ... on Repository {
       name
       url
       description
+      owner { login }
+      labels(first: 100) {
```

Comment:
We should add pagination as we have in the issues query, as this is omitting lot of repositories.

Reply:
I'm curious what this might look like, since the `useDecoratedRepos` hook that implements this query is used only to populate repos that are already added. We;re just not paginating the labels on these repos we know we already want to query. I've created [AUT-92](https://linear.app/issue/AUT-92/add-pagination-to-repo-query) to discuss this further